js/ui/scheduler/ui.scheduler.js

```diff
@@ -1060,6 +1060,35 @@ const Scheduler = Widget.inherit({
         return this._calculateTimezoneByValue(this.option('timeZone'), date);
     },
 
+    _getDaylightOffsetByCustomTimezone: function(startDate, endDate) {
+        return this._getTimezoneOffsetByOption(startDate) - this._getTimezoneOffsetByOption(endDate);
+    },
+
+    _getDaylightOffsetByAppointmentTimezone: function(startDate, endDate, appointmentTimezone) {
+        return this._calculateTimezoneByValue(appointmentTimezone, startDate) - this._calculateTimezoneByValue(appointmentTimezone, endDate);
+    },
+
+    _correctDateByDaylightOffsets: function(originalStartDate, date, startDateTimezone) {
+        const daylightOffsetByOption = this._getDaylightOffsetByCustomTimezone(originalStartDate, date);
+        const daylightOffsetByAppointment = this._getDaylightOffsetByAppointmentTimezone(originalStartDate, date, startDateTimezone);
+        const diff = daylightOffsetByOption - daylightOffsetByAppointment;
+
+        return new Date(date.getTime() - diff * toMs('hour'));
```

Comment:
Lord Howe Island
Yangon

These timezones have hours shift with the minute draw. Does this code support this kind of altitude?

Reply:
I checked the display of the appointment in this timezone, everything works as expected